stable/grafana/values.yaml

```diff
@@ -154,6 +153,35 @@ persistence:
   # subPath: ""
   # existingClaim:
 
+initChownData:
```

Comment:
is the `chownDataImage` section still needed?

Reply:
Is not needed